style.css

```diff
@@ -1 +1,345 @@
 /*  Add your styling here */
+
+body {
+  background-color: #f0d7d2;
+  color: #4285f4;
+  font-family: "Karla", sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header {
+  background-color: rgb(255, 255, 255);
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 80px;
+}
+
+div h1 {
+  padding-left: 110px;
+}
+
+h1 {
+  font-style: italic;
+  font-weight: lighter;
+  font-family: "Vollkorn", serif;
+  position: relative;
+  color: rgb(189, 150, 52);
+}
+
+.logo {
+  width: 80px;
+  max-width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0px;
+}
+
+.nav-menu {
+  display: block;
+  position: fixed;
+  top: 30px;
+  right: 0px;
+  z-index: 1;
+  user-select: none;
+}
+
+.nav-menu input {
+  display: block;
+  width: 40px;
+  height: 30px;
+  position: absolute;
+  top: -8px;
+  right: 9px;
+  cursor: pointer;
+  opacity: 0;
+  z-index: 2;
+}
+
+.menu-box {
+  padding-right: 15px;
+}
+
+.nav-menu .hamburger {
+  display: block;
+  position: relative;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  background-color: #ddbc8e;
+  border-radius: 3px;
+  z-index: 1;
+  transform-origin: 4px 0px;
+  transition: transform 0.5s cubic-bezier(0.77, 0.2, 0.05, 1),
+    background-color 0.5s cubic-bezier(0.77, 0.2, 0.05, 1), opacity 0.55s ease;
+}
+
+.nav-menu span::before {
+  transform-origin: 0% 0%;
+}
+
+.nav-menu span::after {
+  transform-origin: 0% 100%;
+}
+
+.nav-menu-list {
+  list-style: none;
+  text-align: center;
+  position: absolute;
+  width: 320px;
+  margin: 30px -50px 0px 0px;
+  padding: 5px;
+  padding-top: 50px;
+  background-color: cornsilk;
+  transform-origin: 0% 0%;
+  transform: translate(-100%, 0);
+  transition: transform 300ms cubic-bezier(0.5, 0, 0.5, 1);
+}
+
+.nav-menu-list li {
+  font-size: 25px;
+  padding-top: 40px;
+  padding-right: 0px;
+  padding-bottom: 40px;
+}
+
+.nav-menu input:checked ~ ul {
+  transform: translateX(100%);
+}
+
+.menu-link {
+  text-decoration: none;
+  color: olivedrab;
+}
+
+.menu-link:hover {
+  padding-left: 50px;
+  color: brown;
+}
+
+.text-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-top: 15%;
+}
+
+h2 {
+  padding-top: 10px;
+  margin-bottom: 15px;
+  font-size: 21px;
+  color: #4285f4;
+  text-transform: uppercase;
+}
+
+p {
+  margin-top: 10px;
+  color: rgb(65, 47, 4);
+  font-size: 18px;
+  font-style: oblique;
+  width: 80%;
+}
+
+.text-box p {
+  text-align: center;
+}
+
+.main-image {
+  width: 414px;
+  height: 500px;
+}
+
+.collections {
+  display: flex;
+}
+
+.img-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.collections-img {
+  width: 190px;
+  height: 207px;
```

Comment:
Don't set a width AND a height on your `img` as this will distort them.  Setting the width should be enough, and let the height set itself naturally.

To better control the width of the image:
- wrap it inside a `div`
- give that `div` a width (and an optional height)
- and then just `width: 100%` on the `img` (no height)

You do need your images to be the same ratio / same shape for this to work.